src/BladeHealthIconsServiceProvider.php

```diff
@@ -17,7 +17,17 @@ public function register(): void
         $this->callAfterResolving(Factory::class, function (Factory $factory, Container $container) {
             $config = $container->make('config')->get('blade-healthicons', []);
 
-            $factory->add('healthicons', array_merge(['path' => __DIR__.'/../resources/svg'], $config));
+            $filledConfig = array_merge(['path' => __DIR__.'/../resources/svg/filled'], $config);
+            $filledConfig['prefix'] .= '_f';
```

Comment:
Adjusting these to simply just suffix with `f` might work well too.
Then the config file's default base could be `hi`, or even be left as the current `healthicon`.
If left as it is now and changed to a single letter without underscore then the prefix for each set would be `healthiconf`, `healthiconn` and `healthicono`.

Reply:
I think the shorter base would work better - I think having the underscore would make it more readable since there is little control over the names of the icons